simtools/applications/generate_corsika_histograms.py

```diff
@@ -42,68 +42,70 @@
         The configuration used for generating the histograms.
         It includes information about the bin sizes, the ranges, scale of the plot and units.
         By construction, three major histograms are created to start with:
-         - hist_direction (2D): Directive cosines (x and y) for the incoming photons;
-         - hist_position (3D): position x, position y, and wavelength;
-         - hist_time_altitude (2D): time of arrival and altitude of emission;
-
-        If the argument is not given, the default configuration is generated:
-
-        .. code-block:: console
-            hist_direction:
-              x axis: {bins: 100, scale: linear, start: -1, stop: 1}
-              y axis: {bins: 100, scale: linear, start: -1, stop: 1}
-
-            hist_position:
-              x axis:
-                bins: 100
-                scale: linear
-                start: !astropy.units.Quantity
-                  unit: &id001 !astropy.units.Unit {unit: m}
-                  value: -1000.0
-                stop: &id002 !astropy.units.Quantity
-                  unit: *id001
-                  value: 1000.0
-              y axis:
-                bins: 100
-                scale: linear
-                start: !astropy.units.Quantity
-                  unit: *id001
-                  value: -1000.0
-                stop: *id002
-              z axis:
-                bins: 80
-                scale: linear
-                start: !astropy.units.Quantity
-                  unit: &id003 !astropy.units.Unit {unit: nm}
-                  value: 200.0
-                stop: !astropy.units.Quantity
-                  unit: *id003
-                  value: 1000.0
-            hist_time_altitude:
-              x axis:
-                bins: 100
-                scale: linear
-                start: !astropy.units.Quantity
-                  unit: &id004 !astropy.units.Unit {unit: ns}
-                  value: -2000.0
-                stop: !astropy.units.Quantity
-                  unit: *id004
-                  value: 2000.0
-              y axis:
-                bins: 100
-                scale: linear
-                start: !astropy.units.Quantity
-                  unit: &id005 !astropy.units.Unit {unit: km}
-                  value: 120.0
-                stop: !astropy.units.Quantity
-                  unit: *id005
-                  value: 0.0
+        - hist_direction (2D): Directive cosines (x and y) for the incoming photons;
+        - hist_position (3D): position x, position y, and wavelength;
+        - hist_time_altitude (2D): time of arrival and altitude of emission;
+
+    If the argument is not given, the default configuration is generated:
+
+    .. code-block:: console
+
+        hist_direction:
+            x axis: {bins: 100, scale: linear, start: -1, stop: 1}
+            y axis: {bins: 100, scale: linear, start: -1, stop: 1}
+
+        hist_position:
+            x axis:
+            bins: 100
```

Comment:
The indentation here is now wrong (it was right before).

Reply:
Correct - the indentation was generally wrong before, but now it is inconsistent with what we want to say.

Reply:
@orelgueta, in the generated document it looks good.
<img width="802" alt="Screenshot 2023-11-30 at 14 58 51" src="https://github.com/gammasim/simtools/assets/50831535/de2a20f5-99b9-4f48-9ae0-3318fd0817f3">

Reply:
Ah, I see the tabs missing now :)